modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: Procedure
+[id="deploying-book-info"]
+= Deploying the Bookinfo application
+:context: ossm-pdeploying-book-info
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.15 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+* You have access to the {ocp-short-name} CLI (oc).
+* You have installed the {SMProductName} Operator, and deployed {istio}.
+* You have created the {istio} and IstioCNI resources.
+
+. In the {ocp-product-title} web console, navigate to the *Home* -> *Projects* page.
+
+. Click *Create Project*.
+
+. Enter `bookinfo` in the *Project name* field.
++
+The *Display name* and *Description* fields provide supplementary information and are not required. 
+
+. Click *Create*.
+
+. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-injection=enabled
```

Comment:
Not sure that this will only work if istio resource is named 'default'. Otherwise the label must be istio.io/rev=<revisionName>

Reply:
@FilipB I'm not quite following completely...

Can you show me how the syntax should change?

Reply:
In case you are using Istio revisions, you need to use: `oc label namespace bookinfo  istio.io/rev=<revisionName>`

Reply:
done